packages/fxa-settings/src/pages/ResetPassword/ResetPasswordConfirmed/index.test.tsx

```diff
@@ -27,14 +27,15 @@ describe('ResetPasswordConfirmed', () => {
   });
 
   it('emits the expected metrics on render', () => {
-    render(<ResetPasswordConfirmed isSignedIn />);
+    render(<ResetPasswordConfirmed isSignedIn isSync={false} />);
     expect(usePageViewEvent).toHaveBeenCalledWith(viewName, REACT_ENTRYPOINT);
   });
 
   it('emits the expected metrics when a user clicks `Continue`', () => {
     render(
       <ResetPasswordConfirmed
         isSignedIn
+        isSync={false}
```

Comment:
Will we also need tests for `isSync={true}`? Maybe a todo?

Reply:
Added!